test/integration/client/scram-tests.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
```

Comment:
I'm not sure how this can be integrated into the test run? The role needs to be created with a scram-sha-256 password